### PR TITLE
Feature/1569 refactor testing toolkit helm chart

### PR DESCRIPTION
**Pull request by georgi-logodazhki**

### CODE REVIEW

File: ml-testing-toolkit/realm/realm.json [hunk 1]
```diff
@@ -0,0 +1,2293 @@
...
+  "webAuthnPolicyPasswordlessAvoidSameAuthenticatorRegister": false,
+  "webAuthnPolicyPasswordlessAcceptableAaguids": [],
+  "users": [
+    {
+      "username" : "hub",
+      "enabled": true,
+      "firstName": "hub",
+      "lastName": "hub",
+      "credentials" : [{ "type" : "password", "value" : "hub" }],
+      "realmRoles": ["offline_access", "uma_auhtorization"],
```
Comment (vijayg10) [inline ml-testing-toolkit/realm/realm.json:466]: And also we should parameterize the password for hub.

**Review by vijayg10 — CHANGES_REQUESTED** [review]

File: ml-testing-toolkit/chart-backend/templates/config.yaml [hunk 2]
```diff
@@ -13,9 +13,9 @@ metadata:
...
-{{- end }}
+  {{- range $k, $v := .Values.config }}
+    {{ $k }}: {{ ($v | toPrettyJson | squote | replace "$mongodb_host" (index $MONGODB "host") | replace "$mongodb_port" (index $MONGODB "port") | replace "$auth_host" (index $API_AUTHENTICATION "host")  | replace "$auth_port" (index $API_AUTHENTICATION "port") | replace "$auth_issuer_path" (index $API_AUTHENTICATION "paths" "issuer") | replace "$auth_token_iss_path" (index $API_AUTHENTICATION "paths" "tokenIss") | replace "$auth_embedded_certificate" (index $API_AUTHENTICATION "embeddedCertificate") | replace "$connection_manager_host" (index $CONNECTION_MANAGER "host") | replace "$connection_manager_port" (index $CONNECTION_MANAGER "port") | replace "$release_name" $RELEASE_NAME)  }}
+  {{- end }}
```
Comment (vijayg10) [inline ml-testing-toolkit/chart-backend/templates/config.yaml]: Are you sure, these lines are with correct indentation?

File: ml-testing-toolkit/chart-backend/templates/config.yaml [hunk 3]
```diff
@@ -28,6 +28,6 @@ metadata:
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 data:
-{{- range $k, $v := .Values.extraEnvironments }}
-  {{ $k }}: {{ ($v | toPrettyJson | squote | replace "$release_name" $RELEASE_NAME)  }}
-{{- end }}
+  {{- range $k, $v := .Values.extraEnvironments }}
+    {{ $k }}: {{ ($v | toPrettyJson | squote | replace "$release_name" $RELEASE_NAME)  }}
+  {{- end }}
```
Comment (vijayg10) [inline ml-testing-toolkit/chart-backend/templates/config.yaml]: Are you sure, these lines are with correct indentation?

File: ml-testing-toolkit/chart-backend/templates/config.yaml (hunk 2 above)
Comment (vijayg10) [reply ml-testing-toolkit/chart-backend/templates/config.yaml]: I think those are ok

File: ml-testing-toolkit/chart-backend/templates/config.yaml (hunk 3 above)
Comment (vijayg10) [reply ml-testing-toolkit/chart-backend/templates/config.yaml]: I think those are ok

**Review by vijayg10 — APPROVED** [review]

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**